docs/getting-started/building-from-source.md

```diff
@@ -38,6 +38,8 @@ APP_CONFIG_ECM_HOST="<URL>"
 APP_CONFIG_PLUGIN_AOS=false
 APP_CONFIG_PLUGIN_CONTENT_SERVICE=true
 APP_CONFIG_PLUGIN_FOLDER_RULES=true
+APP_CONFIG_ENABLE_MOBILE_APP_SWITCH=true
```

Comment:
again, why this is set as default variables for each setup?

Reply:
I have removed the configuration variables from this file as well